packages/atomic/src/pages/accessibility-a.html

```diff
@@ -0,0 +1,401 @@
+<!DOCTYPE html>
```

Comment:
This page is almost exactly like `index.html`, but a bunch of stuff was removed.

Reply:
can we move these in a folder pages/accessibility
maybe they could share a stylesheet too

Reply:
I think the goal is for Deque to have access to some prebuilt page so that they can do their test.

As such, I don't think it makes sense to put them in the repo: I don't really see how it's going to make their lives easier to test.

How do you plan making it available for them @btaillon  ? 

What if it was just a random repo you created on https://github.com/coveord , then uploaded to Netlify or something ?

Reply:
> As such, I don't think it makes sense to put them in the repo: I don't really see how it's going to make their lives easier to test.

The top reason I added it here was for review, however.

It doesn't make their lives easier, but it may make ours. When testing, I neglected to test the emphasized section, result rating and unevenly sized images because they weren't in `index.html`. `accessibility-b.html` covers all of those though (since it uses a commerce org), so maybe we could just add some version of that one to the repo and skip the other ones.

Reply:
> can we move these in a folder pages/accessibility
> maybe they could share a stylesheet too

They don't exactly have the same stylesheet, I prefer to keep them separate for now.